docs/caching2.md

```diff
@@ -0,0 +1,97 @@
+## Caching
+
+There are two different layers of caching on Artsy.net, namely:
+
+- Caching at the Relay query level, which happens as users navigate from page to page on the client. This is an entirely client-side cache, and typically configured via the route. We will refer to this going forward as 'Relay caching'.
+- Caching at the [Metaphysics CDN](https://github.com/artsy/metaphysics/blob/1b3940a2387ab0bf308c2ca7b597e40fad015b4d/docs/cdn.md) level - which has the effect of caching the entire response for a given query. We will refer to this going forward as 'CDN caching'.
+
+For both of these caching layers, there are several different configurations that can be used/combined to achieve desired behavior.
+
+### General Cache Behavior
+
+#### Logged out:
+
+By default, we typically cache (at both layers) all queries for logged-out visitors, unless otherwise opted-out with `cacheConfig: { force: true }` or `serverCacheTTL: 0`. There are also certain client-side queries made using Relay's `fetchQuery` - which implicitly _adds_ a `force: true` configuration to the query, thus skipping the cache as well. For queries made using `fetchQuery` - you need to add the `force: false` directive explicitly in order to cache. The important thing to remember is: **everything is cached by default unless opted-out**.
```

Comment:
... or `serverCacheTTL: 0`, set on the route.

Reply:
Yes, but that's listed as one of the opt-outs a couple of sentences prior, so figured there's no need to list the opt-outs.

Although maybe the better clarification is to mention up here the nuance of `serverCacheTTL: 0` - that it only applies to the CDN caching layer - Relay caching is still in effect for these (whereas Relay caching is _skipped_ for `force: true`)

Reply:
The first time `serverCacheTTL` is mentioned is here, so thinking about this from the perspective of a user who has never been exposed to caching before, they'll see `serverCacheTTL` and not understand. Hence the idea that we add the ", on the route" to clarify

Reply:
(sorry, i forgot to put that first comment in quotes to indicate i was suggesting a copy update)